Modules/Quotas/test/Quotas.Application.Tests/Tests/Tiers/GetTierById/HandlerTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Backbone.Modules.Quotas.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Quotas.Application.Tests.TestDoubles;
+using Backbone.Modules.Quotas.Application.Tiers.Queries.GetTierById;
+using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
+using Backbone.Modules.Quotas.Domain.Aggregates.Metrics;
+using Backbone.Modules.Quotas.Domain.Aggregates.Tiers;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Quotas.Application.Tests.Tests.Tiers.GetTierById;
+public class HandlerTests
```

Comment:
You should add a more complex test case with multiple quotas, some of which have the same metric, and check that the metrics of the quotas in the DTO are filled correctly.